src/engine/engine.rs

```diff
@@ -51,6 +51,12 @@ pub trait Dev: Debug {
 pub trait Filesystem: HasName + HasUuid {
     /// Rename this filesystem.
     fn rename(&mut self, name: &str) -> ();
+
+    /// Get this filesystem's mount point
+    fn mountpoint(&self) -> &String;
+
+    /// Get this filesystem's quota
+    fn quota(&self) -> &Bytes;
```

Comment:
why are we adding these when a previous commit removed the dbus bits for these, and made the comment they are scheduled for a future release?